xmake/plugins/project/vstudio/impl/vs201x.lua

```diff
@@ -592,11 +593,6 @@ function make(outputdir, vsinfo)
     -- clear local cache
     _clear_cache()
 
-    -- restore previous config data
-    if configdata then
-        io.writefile(config.filepath(), configdata)
-    end
```

Comment:
Why? xmake.conf maybe has been changed, it's not localcache("config")

Reply:
xmake.conf is no longer cleared since those lines got removed:
```lua
    config.clear()
    config.save()
```

so its content is not overwrited

Reply:
ok